__tests__/app.test.js

```diff
@@ -173,3 +169,15 @@ describe("GET /api/reviews/:review_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/reviews/:review_id/comments", () => {
+  it("201: should accept a request body with correct properties and respond with the posted comment", () => {
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send({ username: "bainesface", body: "Time to tickle." })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toEqual("Time to tickle.");
+      });
+  });
+});
```

Comment:
Nice one with the first happy path test here, we also need to consider some more circumstances:
- Lets make sure our code ignores unnecessary properties on the input object
- Lets consider the sad path: what can go wrong with this? I would say there's quite a lot to consider here. The review_id and the input body could have multiple different issues which we need to consider.